Source/WebKit/SourcesGTK.txt

```diff
@@ -41,6 +41,7 @@ NetworkProcess/glib/DNSCache.cpp
 NetworkProcess/glib/WebKitCachedResolver.cpp
 NetworkProcess/glib/WebKitOverridingResolver.cpp
 
+NetworkProcess/NetworkDataTaskDataURL.cpp
```

Comment:
Since the directory hierarchy is different, please change the insert location and add a line break.